t/unit/backends/test_azureblockblob.py

```diff
@@ -0,0 +1,125 @@
+from __future__ import absolute_import, unicode_literals
+
+import contextlib
+import os
+
+import pytest
+from case import Mock, call, patch, skip
+
+from celery.backends import azureblockblob
+from celery.backends.azureblockblob import AzureBlockBlobBackend
+from celery.exceptions import ImproperlyConfigured
+
+MODULE_TO_MOCK = "celery.backends.azureblockblob"
+
+
+@contextlib.contextmanager
+def patch_client():
```

Comment:
I am not sure, but I don't think you need this context manager. `patch` will work as you require, see this [example](https://github.com/celery/celery/pull/4666/files#diff-5cb790f5c0c0aea16658640ccd102c19R149) of mocking an instance method.

Reply:
Great catch! Done in 16ce838.